config/vufind/EDSRecordDataFormatter.ini

```
@@ -0,0 +1,61 @@
+; If you want to add fields that are not included in the drivers getItems method
+; you can copy RecordDataFormatter.ini into this file and adapt the settings for EDS.
+
+; To add filters as in [ItemGlobalFilter] of EDS.ini for single contexts (e.g. core, result-list)
+; a section [<context>_ItemFilter] can be used.
+[core_ItemFilter]
+;excludeLabel[] = 'Availability'
+;includeLabel[] = 'Availability'
+;excludeGroup[] = 'URL'
+;includeGroup[] = 'URL'
+
+[result-list_ItemFilter]
+;excludeLabel[] = 'Availability'
+;includeLabel[] = 'Availability'
+;excludeGroup[] = 'URL'
+;includeGroup[] = 'URL'
+
+; This section is used to set global default options for items.
+[GlobalItemDefaultOptions]
+; If a search link should set on the elements of items (default is false)
```

Missing word, or am I misunderstanding?

```suggestion
; If a search link should be set on the elements of items (default is false)
```